task2/main.py

```diff
@@ -1,16 +1,22 @@
+from configparser import ConfigParser
+
 from argument_parser import arg_pars
 from interface_manager import protocol, InterfaceManager
 from utils import get_logger
 
 
 def main(**kwargs) -> None:
+    # config
+    conf = ConfigParser()
+    conf.read("/home/patrik/Documents/patprotasks/task1/flask_app_config.ini")
```

Comment:
@patrikprovaznik dont use absolute path in the program.. this changes based on the user.. either read it from user input or use absolute path if the project is suppose to contain this in some resources folder